src/config/rds_mysql.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "RDS",
+  "displayName": "Amazon RDS",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/rds/instance/%{DATA:resource_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/rds/instance/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'rds'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, resource_id])"
```

Comment:
I missed that in Bartek's PR in the end - could you add `priority` to this rule and set it to 1 or 10, and then reuse the aws.arn in dt.source_entity calculation? It probably has minor impact, but still spares us one re-defintion and one re-calculation of the arn. Something like this should work:

```
{
          "key": "aws.arn",
          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, resource_id])",
          "priority": 10
        },
        {
          "key": "dt.source_entity",
          "pattern": "dt_meid_rds_v2(aws.arn))"
        },       
```

Reply:
I think in Bartek's PR there was a slight difference, i.e meid input was little different from ARN. Actually we have been looking at this in the office :)

Reply:
Ok, great - I just remembered I wanted to check if this was same or two different, but never did in the end. So just here then, here it's two same ones.

Reply:
Done